Source/WebKit/NetworkProcess/ServiceWorker/WebSWServerConnection.cpp

```diff
@@ -765,6 +767,47 @@ void WebSWServerConnection::retrieveRecordResponseBody(WebCore::BackgroundFetchR
     });
 }
 
+void WebSWServerConnection::addCookieChangeSubscriptions(WebCore::ServiceWorkerRegistrationIdentifier registrationIdentifier, Vector<WebCore::CookieChangeSubscription>&& subscriptions, ExceptionOrVoidCallback&& callback)
+{
+    RefPtr registration = protectedServer()->getRegistration(registrationIdentifier);
+    if (!registration) {
+        RELEASE_LOG_ERROR(CookieStore, "No registration");
```

Comment:
The logging should likely mention the class/function name. `RELEASE_LOG_ERROR` doesn't do that for you, which is why we usually wrap `RELEASE_LOG*` calls in a new macro.